<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RcfServer</class>
 <widget class="QDialog" name="RcfServer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>879</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RcfServer</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="statelabel">
       <property name="text">
        <string>服务器未开启</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearBtn">
       <property name="text">
        <string>清空列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="text">
        <string>开启服务器</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="text">
        <string>停止服务器</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
